src/styles.css

```diff
@@ -1,3 +1,167 @@
+html {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
 body {
-  background-color: blueviolet;
+  flex: 1;
+  color: #011936;
+  display: flex;
+  flex-direction: column;
+  font-family: 'Raleway', sans-serif;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+h1, h2, h3, h4 {
+  font-family: 'Quicksand', sans-serif;
+}
+
+nav {
+  background-color: #E0E0E0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+  border-bottom: 1px solid #011936;
+  padding: 0 20px;
+  width: 100%;
+}
+
+h1 {
+  margin: 0;
+  color: #Ef5D60;
+}
+
+main {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  flex: 1;
+  width: 100%;
+}
+
+main > section {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  width: 100%;
+}
+
+button {
+  background-color: transparent;
+  border-style: none;
+  cursor: pointer;
+  display: flex;
+  justify-content: space-around;
+  padding: 8px;
+}
+
+.nav-bar__nav-tabs {
+  display: flex;
+  justify-content: space-around;
+  flex-direction: row;
+  align-items: center;
+}
+
+.nav-bar__nav-tabs > button {
+  font-family: 'Raleway', sans-serif;
+  font-size: 16px;
+  font-weight: 700;
+  height: 50px;
+  align-items: center;
+  width: 120px;
+}
+
+.nav-bar__nav-tabs > button:hover {
+  border-bottom: 4px solid #Ef5D60;
+}
+
+.nav-bar__home-search {
+  display: flex;
+  align-items: center;
+}
+
+.search-bar {
+  display: flex;
+  justify-content: end;
+  align-items: center;
+}
+
+.search-bar__search-label {
+  margin-right: 8px;
+}
+
+.search-ingredient-glide__images-container {
+  display: flex;
+  gap: 10px;
+  overflow-x: scroll;
+  width: 100%;
+}
+
+.featured__featured-recipe-image-container {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  overflow: hidden;
+  position: relative;
+}
+
+.featured__featured-recipe-image {
+  position: absolute;
+  top: -50%;
+  width: 100%;
+}
+
+.images-container__button {
+  align-items: center;
+  display: flex;
+  height: 200px;
+  justify-content: center;
+  min-width: 200px;
+  overflow: hidden;
+  position: relative;
+}
+
+.images-container__ingredient-name {
+  border: 1px solid #011936;
+  background-color: white;
+  padding: 6px;
+  text-align: center;
+  width: 80px;
+  z-index: 1;
+}
+
+.images-container__image{
+  border: 4px solid #fff;
+  height: 400px;
+  position: absolute;
+}
+
+.featured__search-ingredient-glide {
+  padding: 0 10px 10px;
+}
+
```

Comment:
As our CSS gets longer, we might want to add sections to it via comments, for example, 
*/--------- Home View ---------/*